Tests/DangerSwiftLintTests/DangerSwiftLintTests.swift

```diff
@@ -21,6 +21,30 @@ class DangerSwiftLintTests: XCTestCase {
         XCTAssertNotEqual(executor.invocations.dropFirst().count, 0)
     }
 
+    func testExecutesSwiftLintWithConfigWhenPassed() {
+        let configFile = "/Path/to/config/.swiftlint.yml"
```

Comment:
If this is a continuous test - shouldn't we just bundle a `.swiftlint.yml` mock in the repo ?

Reply:
Nah, this is just needed to check if the argument with this path was added.